RevenueCatUI/Templates/V2/Components/Stack/FlexHStack.swift

```diff
@@ -65,26 +65,23 @@ struct FlexHStack: View {
                 ForEach(0..<componentViewModels.count, id: \.self) { index in
                     ComponentsView(componentViewModels: [self.componentViewModels[index]], onDismiss: self.onDismiss)
                     if index < self.componentViewModels.count - 1 {
-                        Spacer()
+                        Spacer(minLength: 0)
                     }
                 }
 
             case .spaceAround:
-                Spacer()
                 ForEach(0..<componentViewModels.count, id: \.self) { index in
+                    Spacer(minLength: 0)
                     ComponentsView(componentViewModels: [self.componentViewModels[index]], onDismiss: self.onDismiss)
-                    if index < self.componentViewModels.count - 1 {
-                        Spacer()
-                    }
+                    Spacer(minLength: 0)
```

Comment:
Fixed the `spaceAround` flex layout: each items has a spacer around it, so there is two spacers between items and one at the start/end.